lib/commands/start.js

```diff
@@ -59,8 +60,20 @@ class StartCommand extends Command {
                     });
                 }).then(() => {
                     if (!argv.quiet) {
+                        const isInstall = argv._ && argv._.length && argv._[0] === 'install';
+                        let adminUrl = instance.config.get('admin.url') ? instance.config.get('admin.url') : instance.config.get('url');
```

Comment:
you should just be able to do `let adminUrl = instance.config.get('admin.url') || instance.config.get('url');` here

Reply:
👍